from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_Account(object):
    def setupUi(self, Account):
        Account.setObjectName("Account")
        Account.resize(553, 433)
        self.load_picture_button = QtWidgets.QPushButton(Account)
        self.load_picture_button.setGeometry(QtCore.QRect(0, 240, 113, 41))
        self.load_picture_button.setObjectName("load_picture_button")
        self.picture_label = QtWidgets.QLabel(Account)
        self.picture_label.setGeometry(QtCore.QRect(0, 60, 231, 161))
        self.picture_label.setObjectName("picture_label")
        self.login_label = QtWidgets.QLabel(Account)
        self.login_label.setGeometry(QtCore.QRect(0, 10, 551, 31))
        self.login_label.setObjectName("login_label")
        self.email_label = QtWidgets.QLabel(Account)
        self.email_label.setGeometry(QtCore.QRect(0, 310, 60, 31))
        self.email_label.setObjectName("email_label")
        self.email_edit = QtWidgets.QLineEdit(Account)
        self.email_edit.setGeometry(QtCore.QRect(60, 310, 181, 31))
        self.email_edit.setObjectName("email_edit")
        self.delete_acc_button = QtWidgets.QPushButton(Account)
        self.delete_acc_button.setGeometry(QtCore.QRect(10, 370, 131, 51))
        self.delete_acc_button.setObjectName("delete_acc_button")
        self.text_notes_button = QtWidgets.QPushButton(Account)
        self.text_notes_button.setGeometry(QtCore.QRect(360, 100, 181, 101))
        self.text_notes_button.setObjectName("text_notes_button")
        self.table_notes_button = QtWidgets.QPushButton(Account)
        self.table_notes_button.setGeometry(QtCore.QRect(360, 210, 181, 101))
        self.table_notes_button.setObjectName("table_notes_button")
        self.calendar_notes_button = QtWidgets.QPushButton(Account)
        self.calendar_notes_button.setGeometry(QtCore.QRect(360, 320, 181, 101))
        self.calendar_notes_button.setObjectName("calendar_notes_button")
        self.types_of_the_notes_label = QtWidgets.QLabel(Account)
        self.types_of_the_notes_label.setGeometry(QtCore.QRect(360, 70, 181, 21))
        self.types_of_the_notes_label.setObjectName("types_of_the_notes_label")

        self.retranslateUi(Account)
        QtCore.QMetaObject.connectSlotsByName(Account)

    def retranslateUi(self, Account):
        _translate = QtCore.QCoreApplication.translate
        Account.setWindowTitle(_translate("Account", "Dialog"))
        self.load_picture_button.setText(_translate("Account", "Load picture"))
        self.picture_label.setText(_translate("Account", "<html><head/><body><p align=\"center\"><span style=\" font-size:24pt;\">Your picture (.png)</span></p></body></html>"))
        self.login_label.setText(_translate("Account", "<html><head/><body><p align=\"center\"><span style=\" font-size:24pt;\">Login</span></p></body></html>"))
        self.email_label.setText(_translate("Account", "<html><head/><body><p align=\"center\"><span style=\" font-size:14pt;\">Email:</span></p></body></html>"))
        self.email_edit.setText(_translate("Account", "some_mail@mail.ru"))
        self.delete_acc_button.setText(_translate("Account", "Delete account"))
        self.text_notes_button.setText(_translate("Account", "Text notes"))
        self.table_notes_button.setText(_translate("Account", "Table notes"))
        self.calendar_notes_button.setText(_translate("Account", "Calendar notes"))
        self.types_of_the_notes_label.setText(_translate("Account", "<html><head/><body><p align=\"center\"><span style=\" font-size:18pt; text-decoration: underline;\">Types of the notes</span></p></body></html>"))
